src/platform/unittest/DataPathTest.cpp

```diff
@@ -396,6 +452,27 @@ struct CxPlatDataPath {
     uint32_t GetSupportedFeatures() const noexcept { return CxPlatDataPathGetSupportedFeatures(Datapath); }
 };
 
+#ifdef QUIC_USE_RAW_DATAPATH
+extern "C" {
```

Comment:
If it's for just a single function, you don't need the `{` after the `extern "C"`.

Reply:
Changed to just be static